mono/utils/mono-signal-handler.h

```diff
@@ -78,25 +78,36 @@
  */
 
 #ifdef HOST_WIN32
-#define MONO_SIG_HANDLER_SIGNATURE(ftn) ftn (int _dummy, EXCEPTION_POINTERS *_info, void *context)
```

Comment:
The only difference in these two blocks was already factored into MONO_SIG_HANDLER_INFO_TYPE. They were the same otherwise.